horreum-web/src/domain/tests/AllTests.tsx

```diff
@@ -394,22 +372,25 @@ export default function AllTests() {
     const teams = useSelector(teamsSelector)
     const isAuthenticated = useSelector(isAuthenticatedSelector)
     const [rolesFilter, setRolesFilter] = useState<Team>(ONLY_MY_OWN)
+
     const [loading, setLoading] = useState(false)
     const [limit, setLimit] = useState(20)
     const [page, setPage] = useState(1)
     const [direction] = useState<SortDirection>("Ascending")
     const pagination = useMemo(() => ({ page, limit, direction, folder }), [page, limit, direction, folder])
     const [count, setCount] = useState(0)
 
+    const [folders, setFolders] = useState<string[]>([])
+
     const loadTests = () => {
-        setLoading(true)
         fetchTestsSummariesByFolder(alerting, SortDirection.Ascending, pagination.folder, pagination.limit, pagination.page, rolesFilter.key, )
-            .then(summary => {setTests(summary.tests?.map(t => mapTestSummaryToTest(t)) || [])
+            .then(summary => {
+                setTests(summary.tests?.map(t => mapTestSummaryToTest(t)) || [])
                 if (summary.count) {
                     setCount(summary.count)
                 }
```

Comment:
With commit https://github.com/Hyperfoil/Horreum/pull/1290/commits/9781289787ac737f8b749e60680db44e9b1c1a1f I had to re-add this `setCount` call otherwise the pagination would not work as expected (always set to 0)

Reply:
ok